lib/tasks/ci/licenses.rake

```diff
@@ -9,15 +9,15 @@ namespace :ci do
     desc 'Check license compliance of dependencies'
     task :compliance do
       STDOUT.puts 'Checking license compliance'
-      unless system("BUNDLE_GEMFILE='Gemfile.on_prem' lib/threescale_license_finder/bin/threescale_license_finder")
+      unless system("BUNDLE_GEMFILE='Gemfile' lib/threescale_license_finder/bin/threescale_license_finder")
         STDERR.puts "*** License compliance test failed  ***"
         exit 1
       end
     end
     desc 'Generates a report with the dependencies and their licenses'
     task :report do
       STDOUT.puts 'Generating report...'
-      exec("BUNDLE_GEMFILE='Gemfile.on_prem' lib/threescale_license_finder/bin/threescale_license_finder report --format=xml > #{Rails.root.join('doc/licenses/licenses.xml')}")
+      exec("BUNDLE_GEMFILE='Gemfile' lib/threescale_license_finder/bin/threescale_license_finder report --format=xml > #{Rails.root.join('doc/licenses/licenses.xml')}")
```

Comment:
This is going to generate the report with "saas" gems, do we want that?

Reply:
Honestly, I have no idea. But you can't generate licenses for what you don't install and here we are not installing the other gemfile.
Once https://github.com/3scale/porta/pull/44 is merged the "on prem" will be the main gemfile and it should all work.